scripts/catalog-tests/resources.json

```diff
@@ -0,0 +1,314 @@
+{
+    "DiscoverableCatalogSnapshot1": {
+        "files": [
+            {
+                "count": 1,
+                "dcat:byteSize": 24941551,
+                "dcat:mediaType": "csv"
+            },
+            {
+                "count": 1,
+                "dcat:byteSize": 2988291633,
+                "dcat:mediaType": "h5ad"
+            }
+        ],
+        "counts": {
+            "files": 2,
+            "donors": 120,
+            "samples": 100
+        },
+        "samples": {
+            "genus": [
+                "Homo sapiens"
+            ],
+            "disease": [
+                "asymptomatic COVID-19 infection",
+                "critical COVID-19 infection",
+                "mild COVID-19 infection",
+                "moderate COVID-19 infection",
+                "normal",
+                "severe COVID-19 infection"
+            ]
+        },
+        "storage": [
+            {
+                "region": "us-central1",
+                "cloudPlatform": "gcp",
+                "cloudResource": "bigquery"
+            },
+            {
+                "region": "us-east4",
+                "cloudPlatform": "gcp",
+                "cloudResource": "firestore"
+            },
+            {
+                "region": "us-central1",
+                "cloudPlatform": "gcp",
+                "cloudResource": "bucket"
+            }
+        ],
+        "dct:title": "Discoverable Catalog Snapshot 1",
+        "dct:issued": "2021-05-20T11:31:20.407000Z",
+        "dct:creator": {
+            "foaf:name": "Human Cell Atlas"
+        },
+        "contributors": [
+            {
+                "email": null,
+                "laboratory": "Wellcome Genome Campus",
+                "contactName": "Waradon,,Sungnak",
+                "institution": "Wellcome Sanger Institute",
+                "projectRole": "experimental scientist",
+                "correspondingContributor": false
+            },
+            {
+                "email": null,
+                "laboratory": "MRC Cambridge Stem Cell Institute,",
+                "contactName": "Fernando,,Calero-Nieto",
+                "institution": "Wellcome - University of Cambridge",
+                "projectRole": "experimental scientist",
+                "correspondingContributor": false
+            },
+            {
+                "email": null,
+                "laboratory": "Molecular Immunity Unit, Department of Medicine",
+                "contactName": "Zewen,,Kelvin Tuong",
+                "institution": "University of Cambridge",
+                "projectRole": "experimental scientist",
+                "correspondingContributor": false
+            },
+            {
+                "email": "wteh@ebi.ac.uk",
```

Comment:
Even though this info is public, I wonder if we should replace these with dummy emails, that way GitHub bots won't use this for spam.

Reply:
That's a good point, I've updated the emails and the names to be randomly generated